polynote-kernel/src/main/scala/polynote/kernel/LocalKernel.scala

```
@@ -192,7 +211,7 @@ class LocalKernel private[kernel] (
     } yield (cell, interpreter, State.id(id, prevState))
   }.provideSomeLayer[BaseEnv with GlobalEnv with CellEnv](CurrentTask.none).map {
     result => Option(result)
-  }.catchAll(_ => ZIO.succeed(None)).someOrFailException  // TODO: need a real OptionT
+  }.catchAll(err => Logging.error(err).as(None)).someOrFailException  // TODO: need a real OptionT
```

ooh, were we swallowing some errors? 😬